services/maintenance/Dockerfile

```diff
@@ -0,0 +1,21 @@
+#FROM nbgallery/jupyter-alpine:7.8.4
+FROM jupyter/base-notebook:python-3.7.3 AS base
+
+
+# COPY --chown=jovyan:users scripts/docker/healthcheck_curl_host.py /healthcheck/healthcheck_curl_host.py
```

Comment:
you should use the same healthcheck as the one used with the jupyter notebook service.
I think it is important to know if the service really works or is responsive.